conf/eslint.json

```diff
@@ -75,6 +75,7 @@
         "no-process-env": 0,
         "no-process-exit": 2,
         "no-proto": 2,
+        "no-reassign-constant": 0,
```

Comment:
I think `no-const-reassign` would be better. Most rules with "assign" have that word last, and "const" makes it very clear we're dealing with `const` declarations.